### ISSUE TITLE
Face Extract Slow on 4090

### ISSUE THREAD

**Issue by pjthefey** [iperov/DeepFaceLab#5696]
## Expected behavior

I'm extracting faces from a DST video and I expected it to go much faster than it is given my system specs

## Actual behavior

It's extracting but pretty slow. It took about 45 seconds to even start the process.   Over 3 hours for 21k frames. 1.9it/s

## Steps to reproduce

Stopped and restarted the extract. 

Edit: 

It's slightly faster 5 minutes later but not by much.   Here's the Screenshot. 

![image](https://github.com/iperov/DeepFaceLab/assets/104025658/f8d305b1-da6a-4dea-9a79-4d9be2e7561a)

**Comment by Cioscos** [comment]
Yes the process is pretty slow even with 4090, it's not your problem, it's the extractor architecture.
